resources/js/Components/Navigation.svelte

```diff
@@ -1,19 +1,163 @@
+        >
+            <div class="flex items-center justify-between">
+                <a use:inertia href={route("home")}>
+                    <div
+                        class="flex flex-row h-full justify-start items-center"
+                    >
+                        <Logo />
+                        <h1 class="shrink-0 text-3xl px-2">Abidan Archive</h1>
+                    </div>
+                </a>
+                <!-- Mobile menu button -->
+                <!-- svelte-ignore a11y-click-events-have-key-events -->
+                <div on:click={toggleNavbar} class="flex md:hidden">
+                    <button
+                        type="button"
+                        class="text-white-100 hover:text-white-400 focus:outline-none focus:text-white-400"
+                    >
+                        <svg
+                            xmlns="http://www.w3.org/2000/svg"
+                            fill="none"
+                            viewBox="0 0 24 24"
+                            stroke-width="1.5"
+                            stroke="currentColor"
+                            class="w-8 h-8"
+                        >
+                            <path
+                                stroke-linecap="round"
+                                stroke-linejoin="round"
+                                d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+                            />
+                        </svg>
+                    </button>
                 </div>
-            </a>
-        </div>
+            </div>
+
+            <!-- Mobile Menu open: "block", Menu closed: "hidden" -->
+            <div
+                class="flex-col mt-8 lg:text-xl space-y-4 md:flex md:space-y-0 md:flex-row md:items-center md:space-x-6 lg:space-x-10 md:mt-0 {showMenu
+                    ? 'flex'
+                    : 'hidden'}"
+            >
+                <a
+                    class="flex text-white-500 hover:text-abidan-300 "
+                    use:inertia
+                    href={route("home")}
+                >
+                    <div class="px-2">
+                        <svg
+                            xmlns="http://www.w3.org/2000/svg"
+                            fill="none"
+                            viewBox="0 0 24 24"
+                            stroke-width="1.5"
+                            stroke="currentColor"
+                            class="lg:w-7 lg:h-7 w-6 h-6"
+                            ><path
+                                stroke-linecap="round"
+                                stroke-linejoin="round"
+                                d="M2.25 12l8.954-8.955c.44-.439 1.152-.439 1.591 0L21.75 12M4.5 9.75v10.125c0 .621.504 1.125 1.125 1.125H9.75v-4.875c0-.621.504-1.125 1.125-1.125h2.25c.621 0 1.125.504 1.125 1.125V21h4.125c.621 0 1.125-.504 1.125-1.125V9.75M8.25 21h8.25"
+                            /></svg
```

Comment:
I don't know what SVG this is, but let's move it into a component.

Reply:
Moved into hamburger component